gcc/rust/expand/rust-macro-builtins.cc

```diff
@@ -306,4 +306,42 @@ MacroBuiltin::concat (Location invoc_locus, AST::MacroInvocData &invoc)
   return AST::ASTFragment ({node});
 }
 
+AST::ASTFragment
+MacroBuiltin::cfg (Location invoc_locus, AST::MacroInvocData &invoc)
+{
+    // only allow on cfg macros
+    if (invoc.get_path () != "cfg")
+        return AST::ASTFragment::create_error ();
+
```

Comment:
We can remove this bit since the builtin will only be called on invocations of the `cfg` macro

Reply:
@antego This is still present in your latest changes

Reply:
oops, fixed